README.rst

```diff
@@ -1 +1,150 @@
-Stub readme.
+edx_event_bus_kafka
+=============================
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Kafka implementation for Open edX event bus.
+
+Overview (please modify)
+------------------------
+This repository details an implementation of the event bus using Kafka.
+
+Essentially what the event bus does is it processes events from microservices, events can be sent in to the event bus and sent out to be consumed by consumers.
+Events can contain modified data which then send signals to be consumed to their respective consumer via the event bus.
+Implementing the event bus will allow for asynchronous messaging across services which greatly improves efficiency as we don't have to wait for scheduled batch synchronizations.
+
+Event bus will have containers that run management commands using Kubernetes by using a loop to listen for new events and run processing code for each event.
+For deployment purposes, the event bus will use SaaS provider confluent.
```

Comment:
Remove this sentence. The event bus should work with any Kafka deployment (local or hosted). Remember that ideally this app will be usable by folks outside 2u.